TableController/TableController/Sources/TableController.swift

```diff
@@ -248,6 +248,7 @@ public extension TableController {
     }
     
     func tableView(_ tableView: UITableView, didEndDisplaying cell: UITableViewCell, forRowAt indexPath: IndexPath) {
+        guard indexPath.section < self.sectionControllers.count else { return }
```

Comment:
This fixes the crash, but I don't like that it happens. If the sections have been deleted, I don't understand why this is being called for invalid indices.